### ISSUE TITLE
Sample playback causes audio effects to not work in web exports.

### ISSUE THREAD

**Issue by kyanbasu** [godotengine/godot#95991]
### Tested versions

Reproducible in godot 4.3-stable, since it has sample playback as default now.
Not reproducible in Godot 4.2.2

### System information

Windows 10, Godot 4.3-stable, Compatibility

### Issue description

Audio effects applied on busses don't work when `audio/general/default_playback_type.web` is set to `Sample`.
When playback is set to Stream effects work.

### Steps to reproduce

Set any audio effect on bus, notice change by running in editor - everything works as intended.
Then export or remote debug in web and it doesn't work.

### Minimal reproduction project (MRP)

[sample-playback.zip](https://github.com/user-attachments/files/16729487/sample-playback.zip)

**Comment by AThousandShips** [comment]
CC @adamscott

**Comment by adamscott** [comment]
This feature is not supported for now for samples. We could support eventually base effects for the Web platform, but not custom GDScript ones.

**Comment by kyanbasu** [comment]
So why is it default now?

**Comment by adamscott** [comment]
Please read [my blog post](https://godotengine.org/article/progress-report-web-export-in-4-3/) on the subject.

TL;DR: If you're using threads, feel free to revert to streams. But if you're targeting single-threaded exports, streams have huge issues/are unusable, hence the creation of samples. And there's the fact that single-threaded games are the standard on the web, for compatibility and usability reasons.
